tools/pretty-json.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python3
+# SPDX-FileCopyrightText: 2023 Volker Krause <vkrause@kde.org>
+# SPDX-License-Identifier: MIT
+
+# "Pretty" JSON formatting for the long coordinate arrays
+
+import argparse
+import json
+import re
+
+parser = argparse.ArgumentParser(description='Pretty JSON formatting for the Transport API data files.')
+parser.add_argument('filename')
+arguments = parser.parse_args()
+
+# read as JSON and normalize to standard formatting
+with open(arguments.filename, 'r') as f:
+    j = json.loads(f.read())
+s = json.dumps(j, indent = 2)
+
+# fold arrays of scalar values into one line
+s = re.sub(r'\[\n +(\"[A-Za-z-]+\"|[\d\.-]+)', r'[\1', s)
```

Comment:
What do you think about doing this transformation only inside `coverage.area`?

Reply:
We could do that, but that would probably mean writing a full custom JSON serializer, simple post-processing on the JSON string isn't going to be enough for that I suspect.

The difference are the usually very short arrays for Hafas product bitmasks or supported languages/coverage regions for example, splitting those per line doesn't add that much readability/diff-ability.

So unsure whether that's worth the effort?

Reply:
probably not...